serialization/src/include/nrf_802154_spinel_backend.h

```diff
@@ -0,0 +1,80 @@
+/*
```

Comment:
Following directory scheme for platform in `driver` and `sl` I think this file (and other platform files for serialization) should be moved to `serialization/platform/include`